pystiche_papers/ulyanov_et_al_2016/_modules.py

```diff
@@ -49,6 +56,22 @@ def upsample() -> nn.Upsample:
 
 
 class HourGlassBlock(SequentialWithOutChannels):
+    r"""HourGlassBlock from :cite:`ULVL2016`.
+
+    This block comprises a
+    :func:`~pystiche_paper.ulyanov_et_al_2016._modules.downsample`  to half the size
+    using :class:`~torch.nn.AvgPool2d` followed by a processing with ``intermediate``
```

Comment:
How the down- and upsampling is done should be explained in `downsample` and `upsample` respectively.